### DISCUSSION TITLE
Планируете ли вы продолжать разработку Metahealth?

### DISCUSSION THREAD

**Discussion by avaj0** [HowProgrammingWorks/LiveQA#166] [Q&A]
Здравствуйте, репозиторий проекта [Metahealth](https://github.com/meta-edu/Metahealth) не обновлялся в течении последних 11 месяцев.
Планируете ли вы/сообщество развивать проект в дальнейшем?
Спасибо.

**Comment by tshemsedinov** [comment]
Медицинский проект приостановлен на неопределенное время, однако, работы не прекратились, а переведены на другое направление, почти все компоненты будут сделаны, как общие модули технологического стека, одинаковые для многих приложений подобного класса, будь то складские, банковские, транспортные и логистические, реестры и базы данных со всевозможными апи и пользовательскими интерфейсами. См. https://github.com/metarhia/Roadmap Над этим работает команда и каждую неделю мы созваниваемся и координируемся.
